utils/generate_inference_types.py

```diff
@@ -354,7 +354,7 @@ def check_inference_types(update: bool) -> NoReturn:
     for file in INFERENCE_TYPES_FOLDER_PATH.glob("*.py"):
         if file.name in IGNORE_FILES:
             continue
-        content = file.read_text()
+        content = file.read_text().lstrip("\r\n")
```

Comment:
(nit) or even?

```suggestion
        content = file.read_text().lstrip()
```

(should be more robust to encoding issues)